IPython/lib/latextools.py

```diff
@@ -10,6 +10,7 @@
 import shutil
 import subprocess
 from base64 import encodebytes
+from textwrap import wrap as splitstring
```

Comment:
This is kind of non-standard, so I'd prefer if you use simply `wrap` here instead of using an alias.

Reply:
The issue is there is a conflicting keyword named `wrap`, where this is used. I've pushed a commit that use the fully qualified name.

Reply:
Thanks @Carreau ! Didn't really take a step back and figured out a better solution, but was just trying to get it to work. Clearly, using a fully qualified name is the way to go.